jsonnet/main.jsonnet

```diff
@@ -106,6 +106,30 @@ local kp = (import 'kube-prometheus/kube-prometheus.libsonnet') +
                },
              },
              telemeterClient+:: {
+               deployment+: {
+                 spec+: {
+                   template+: {
+                     spec+: {
+                       containers:
+                         std.map(
+                           function(c)
+                             if c.name == 'reload' then
+                               c {
+                                 args: std.map(
+                                   function(a)
+                                     std.strReplace(std.strReplace(a, '--webhook-url=', '--reload-url='), '--volume-dir=', '--watched-dir=')
+                                   ,
+                                   c.args,
+                                 ),
+                               }
+                             else
+                               c,
+                           super.containers,
+                         ),
+                     },
+                   },
+                 },
+               },
```

Comment:
@s-urbaniak for awareness: rather than modifying the "upstream" [mixin](https://github.com/openshift/telemeter/blob/master/jsonnet/telemeter/client/kubernetes.libsonnet), I've decided to override because it avoids opening a third PR for the bump and it would be easier to backport to 4.6 if needed.

Reply:
sounds good, thank you for the heads-up!

side note: the above code snippet looks very scary :see_no_evil: